src/main/java/org/hyperskill/hstest/v6/outcomes/Outcome.java

```diff
@@ -58,9 +58,11 @@ public final String toString() {
 
         if (fullLog.trim().length() != 0) {
             result += "\n\n" +
-                "Below you can see the output of your program during this test\n" +
-                "Notice, that '>' symbol means the start of the input:\n\n";
-
+                "Please find below the output of your program during this failed test.\n";
+            if (fullLog.matches('.*\n>.*')) {
+                result += "The '>' symbol corresponds to input lines.\n";
```

Comment:
If there's a line in the output like this (output and input on the same line):
```
Input action: >info
```
Is this understandable using phrase "input lines"?

Reply:
@vyahhi and also `\n>` seems wrong because the user can write the code like this:
```
System.out.print("Write action: ") // not println !
String str = scanner.nextLine()
```
And the log would be (in case user enters "add") as the following:
```
Write action: >add
```